### ISSUE TITLE
Robot falling when CTRL-C is pressed

### ISSUE THREAD

**Issue by cv2000** [tork-a/minas#81]
Hi,

I have experienced sometimes that the brake does not engage immediately after pressing CTRL-C causing the robot to fall. Have anybody else experienced this or have suggestions for a fix either in drive settings or in the code?
I think it happens during the servoOff routine.

**Comment by 7675t** [comment]
Yes, I have experienced it. But I can't dig it because I don't have hardware anymore.

Ctrl-C calls shutdown() as in

https://github.com/tork-a/minas/blob/4ff346d334078f2958222024accb07499c6d66d9/minas_control/src/minas_hardware_interface.cpp#L143-L150

Just an idea, but it may cause some difference if we change to call servoOff() before reset(). Can you try it?

**Comment by cv2000** [comment]
Thanks!
I have tried it now, but the robot can still fall.
